### DISCUSSION TITLE
GitHub Community Programs 🚀

### DISCUSSION THREAD

**Discussion by ossanna16** [community/community#68410] [New to GitHub]
![454f5e3ad656a3e84a80ea2b39ecef65b36b50dba3790033905ec1d36564a818-1200x630](https://github.com/user-attachments/assets/7191d0da-77a5-4355-ab30-c6666ff09c73)

*Last updated on March 5, 2025.*

According to the [Octoverse 2024](https://octoverse.github.com) report 📊:

* Over 100 million developers are on GitHub
* In 2024, developers around the world made more than 5.2 billion contributions to more than 518 million open source, public, and private projects
* The global community of developers on GitHub is expanding rapidly—and the next generation of developers is getting started on GitHub

This pace of development and collaboration is remarkable, it’s a testament to the power of community, and shows how GitHub is a community of communities. 

### Community conversations
* **💬 [GitHub Community Discussions](https://github.com/orgs/community/discussions/)** (you’re here now, yay! 🎉): A dedicated space for GitHub user self-help, including Q&A, official GitHub announcements, threaded conversations during events like Universe, best practice resources, and a channel for product feedback and feature requests.
* **🧰 [Maintainer Community:](https://github.com/community/maintainers)** A private space for maintainers to connect with peers, preview GitHub features, and help us support the open source community. To learn more and request to join visit: https://maintainers.github.com/.

### [Open source projects](https://opensource.guide/) & contributors
* **💰 [GitHub Sponsors:](https://github.com/sponsors)** Enables you to sponsor the open source software your team has built its business on. Fund the projects that make up your software supply chain to improve its performance, reliability, and stability.
* **⭐ [GitHub Stars:](https://stars.github.com/)** Recognize and lift up the people who inspire and educate your communities with the GitHub Stars program.

### Supporting GitHub user growth
* **✨ [GitHub for Startups:](https://github.com/enterprise/startups)** Helps your startup go from idea to IPO on the world’s largest and most advanced developer platform. Eligible startups receive 20 seats of GitHub Enterprise free for 12 months.
* **👬 [Partner with GitHub:](https://partner.github.com/)** Grow your business with GitHub. Whether you’re building solutions that integrate with GitHub or services that help customers reach their full potential, get the resources and technical support you need.
* **📈 [GitHub Accelerator:](https://accelerator.github.com/)** A 10-week accelerator program providing fellows with an initial sponsorship as well as mentorship from open source leaders and enterprise partners on how to grow their projects and build durable streams of funding.
* **🫂 [DPG Open Source Community Manager Program:](https://socialimpact.github.com/tech-for-social-good/dpg-open-source-community-manager-program)** GitHub Tech for Social Good is running a new Open Source Community Manager Program to support seven digital public goods.

### Events
* **🪩 [GitHub Events:](https://github.com/events)** Connect with the GitHub community at conferences, meetups, and hackathons around the world. Two major events to highlight are [GitHub Universe](https://githubuniverse.com/) and [GitHub Galaxy](https://galaxy.github.com/).

### Learning
* **✍️ [GitHub Skills:](https://skills.github.com/)** Learn how to use GitHub with interactive courses designed for beginners and experts.
* **📓 [GitHub Resources:](https://resources.github.com/)** Resources to help enterprise teams do their best work.
* **📝 [GitHub Docs:](https://docs.github.com/en)** Learn how to use GitHub effectively and collaboratively. You can access GitHub Docs in nine languages. 
* **💻 [GitHub on Microsoft Learn:](https://learn.microsoft.com/en-us/training/github/)** Learn, develop, and master essential Git and GitHub skills and join millions of developers and companies worldwide to build, ship, and maintain software on GitHub - the largest and most advanced development platform in the world.
* **📚 [The ReadME Project:](https://github.com/readme)** The ReadME Project amplifies the voices of the developer community by telling stories about Open Source, AI, DevOps & Automation, Enterprise Scale, Machine Learning, and more.
* **🤲 [All In:](https://allinopensource.org/)** GitHub is a partner of All In, a community dedicated to advancing diversity, equity, and inclusion within open source.
* **⏭️ [GitHub Next:](https://githubnext.com/)** Investigates the future of software development and identifies new approaches to building healthy, productive software engineering teams.
* **🏅 [Partner Enablement and Accreditation:](https://portal.github.partners/#/page/login)** Training resources targeted to GitHub partners, available through the partner portal and our learning management system.

### GitHub Education 
* **🌐 [GitHub Education:](https://education.github.com/)** GitHub Education bridges the gap between coding education and a tech career, and is accessible to everyone globally at no cost. Students can explore and hone their skills with the[ Student Developer Pack](https://education.github.com/pack) and [GitHub Pro](https://docs.github.com/en/get-started/learning-about-github/githubs-plans#github-pro), building a robust portfolio and connecting with a vibrant community. Teachers can streamline their workflow with tools like [GitHub Classroom](https://classroom.github.com/),  facilitating smooth assignment management and progress monitoring. Meanwhile, schools can leverage the [GitHub Campus Program](https://education.github.com/schools) to integrate a rich array of premium GitHub resources into their educational environments, encouraging innovativeness and collaboration.
* **🎒 [GitHub Campus Experts:](https://education.github.com/experts)** A community of student leaders that strive to build diverse and inclusive spaces to learn skills, share their experiences, and build projects together. They can be found across the globe leading in-person and online conferences, meetups, and hackathons, and maintaining open source projects.

We hope that you embrace the power of open source and take part in GitHub’s community programs. Let us know in the comments below which programs you are interested in – or may already be a part of ⬇️

**Comment by melvincwng** [comment]
@ossanna16 Amazing to see the myriad of wonderful opportunities provided by GitHub 🎉

**Reply by ossanna16** [reply]
Thank you @melvincwng 🥇

**Reply by OGherc86** [reply]
Thanks

**Comment by karasowles** [comment]
Thanks for this great overview <3

**Comment by Alcatraz-Dev** [comment]
Thank you @ossanna16 🙏🏻

**Comment by JunLovin** [comment]
Thank you so much @ossanna16 <3

**Reply by PAN740623** [reply]
建议更改

**Comment by usamahussaindev** [comment]
thankyou so much @Anna Ossowski

**Reply by PAN740623** [reply]
更改

**Comment by PAN740623** [comment]
建议更改

**Reply by PAN740623** [reply]
建议更改

**Comment by pichpeyman** [comment]
pich

**Comment by pichpeyman** [comment]
pich

**Comment by S-Oktay-Bicici** [comment]
Thanks :)

**Comment by DTU-Fan** [comment]
Betreff: Zusatzfunktion bei Open-DTU on-Battery

Ich rege an und wünsche mir auch als Ergänzung zum „Dynamic Power Limiter“, eine zusätzliche Funktion bei der Open-DTU on-Battery in der Form, dass eine, oder auch mehrere Steckdose(n) ein- bzw. ausgeschaltet werden können, wenn Überschussstrom erzeugt wird. (Per „IP-Adresse“ und „websend“ im WLAN-Heimnetz).
Die erforderlichen Daten werden ja vom IR-Lesekopf am Zähler per „Power Meter“ im „Dynamic Power Limiter“ weiterverarbeitet, sind also vorhanden.

Als Beispiel: 
Es wird vom Lesekopf ein Stromüberschuss von -200 W gemeldet, (dieser Wert sollte frei wählbar sein), dann sollte nach einer Einschalt-Verzögerung von ca. 300 Sek., damit nicht bei jeder kleinen Änderung ein Einschaltvorgang ausgelöst wird, (auch dieser Wert sollte frei wählbar sein), eine WLAN-Steckdose eingeschaltet werden, (z.B. die „tasmota NOUS A1T“), an dieser Steckdose hängt ein Ladegerät zur Unterstützung der Batterieladung für die Nachteinspeisung. Geht der Wert auf +50 W zurück, erfolgt also ein Bezug aus dem Stromnetz, (dieser Wert sollte frei wählbar sein), dann sollte die Steckdose nach einer Ausschalt-Verzögerung von ca. 300 Sek., damit nicht bei jeder kleinen Änderung ein Ausschaltvorgang ausgelöst wird, (dieser Wert sollte frei wählbar sein), wieder ausgeschaltet werden.

Ich denke, dieses wäre eine sinnvolle Ergänzung zur Nulleinspeisung in der Nacht über den „Dynamic Power Limiter“ und könnte die feste Voreinstellung bei „AC-Ladegerät“ (Huawei R4850G2 an CAN Bus Interface aktiv) ablösen oder erweitern.
Sollte eine solche Funktion bereits verfügbar sein, dann lassen Sie es mich bitte wissen.

Gruß,
Hans-Jürgen

**Comment by NushaMBZ** [comment]
thank you

**Comment by LaLiiSa-28** [comment]
**Thanks a lotttt**

**Comment by SumitKumargiri** [comment]
Thanks a lot

**Comment by mhmtogt** [comment]
Hello sr I am a student but I can't login to github student why please help
me about this problem

Sumitgiri15434 ***@***.***>, 17 Eki 2023 Sal, 15:26 tarihinde
şunu yazdı:

**Comment by ValentinKt** [comment]
Thank you so much @ossanna16 :)

**Comment by sunilnetin** [comment]
Hi, how to connect with Github users to discusses on any code?

**Comment by GaoYingZhang** [comment]
Hi Everyone ,l am from Cambodia .l 'm happy to particpatete in community discussions.

**Reply by ossanna16** [reply]
Hi there @GaoYingZhang and welcome! Please introduce yourself via our official [Introduction Discussion](https://github.com/orgs/community/discussions/23204) for more visibility.

**Comment by mhmtogt** [comment]
Glad to hear it!

9 Kas 2023 Per 21:09 tarihinde GaoYingZhang ***@***.***> şunu
yazdı:

**Comment by Darknessly0** [comment]
Thanks a lot.
